glyphs2fontir/src/source.rs

```diff
@@ -845,12 +860,13 @@ impl Work<Context, WorkId, Error> for FeatureWork {
 
         context.features.set(to_ir_features(
             &font.features,
-            self.font_file_path
-                .canonicalize() // if we got this far, the path must exist
-                .unwrap()
-                .parent()
-                .unwrap() // the path must be in a directory
-                .to_path_buf(),
+            self.font_file_path.as_ref().map(|path| {
+                path.canonicalize()
+                    .unwrap()
+                    .parent()
+                    .unwrap() // the path must be in a directory
```

Comment:
I know this preexists but since we're here ... let's expect rather than unwrap so if this does ever go off we have more clues.